vscode-extension/editor/src/VSCodeTheme.ts

```diff
@@ -0,0 +1,218 @@
+import { MantineThemeOverride } from "@mantine/core";
+
+// For quicker theme devX, use this theme to pass through to the AIConfigEditor instead of
+// the static VSCodeTheme packaged with the editor. We will consolidate later.
+export const VSCODE_THEME: MantineThemeOverride = {
+  defaultGradient: {
+    from: "#ff1cf7",
+    to: "#ff1cf7",
+    deg: 45,
+  },
+
+  globalStyles: () => ({
+    body: {
+      padding: "0 !important",
+      color: "var(--vscode-editor-foreground)",
+    },
+    ".addPromptRow": {
+      borderRadius: "0px",
+    },
+    ".editorBackground": {
+      background: "var(--vscode-editor-background)",
+      margin: "0",
+      maxWidth: "100%",
```

Comment:
Override this from default styles to make full width (otherwise, have white bars on left and right)